src/test/java/com/iota/iri/IXITest.java

```diff
@@ -1,79 +1,82 @@
 package com.iota.iri;
 
 import com.iota.iri.service.dto.AbstractResponse;
-import com.iota.iri.service.dto.IXIResponse;
+import com.iota.iri.service.dto.ErrorResponse;
+import org.hamcrest.CoreMatchers;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Unit tests for {@link IXI}
+ */
```

Comment:
Just that you know, I don't mean to enforce javadocs on unit tests.
I have nothing against it  of course :-)